loadgen/loadgen.cc

```diff
@@ -831,7 +831,7 @@ void PerformanceSummary::Log(AsyncSummary& summary) {
 
   switch (settings.scenario) {
     case TestScenario::SingleStream: {
-      summary("90th percentile latency (ns) : ", latency_target.value);
+      summary("90th percentile latency (ns) : ", target_latency_percentile.value);
```

Comment:
Can you add a TODO to replace "90th" with the real string. (Or use DoubleToString from util.h.)